cli/pcluster/config/validators.py

```diff
@@ -688,3 +688,19 @@ def compute_instance_type_validator(param_key, param_value, pcluster_config):
         errors, warnings = ec2_instance_type_validator(param_key, param_value, pcluster_config)
 
     return errors, warnings
+
+
+def intel_hpc_validator(param_key, param_value, pcluster_config):
```

Comment:
Could you add a test for the validator too? In the `test_validators.py` file.

Reply:
done